Competition2017/src/competition/subsystems/pose/PoseSubsystem.java

```diff
@@ -103,10 +99,16 @@ public double getBreakBaselineMaximumTime() {
         return breakBaselineMaxTime.get();
     }
     
+    public double getIdealShootingRange() {
+        return idealShootingRange.get();
+    }
+    
+    public DistanceSensorPair getLidar() {
```

Comment:
Hmmm... I see what you mean. I can't think of a better solution, so I'd say we should go with this for now.